unstructured/partition/pdf.py

```diff
@@ -420,7 +408,7 @@ def _partition_pdf_or_image_local(
 
     # NOTE(alan): starting with v2, chipper sorts the elements itself.
     if model_name == "chipper":
-        kwargs["sort_mode"] = SORT_MODE_DONT
+        kwargs["sort_mode"] = SortMode.SORT_MODE_DONT
```

Comment:
not related to kwargs, though, right?
otherwise, a reason i'm not enthusiastic about this change is that end users (even using clients in other programming languages) will want to make a call to `partition()` and should just be able to pass a string which doesn't cleanly translate to the python builtin `enum`. as long as there is a ValueError in the right place for a string value... constants are fine IMO.

Reply:
Reverted sort mode enum in [1d1998c](https://github.com/Unstructured-IO/unstructured/pull/1810/commits/1d1998c48d896c9a5f2f1a612be305aa153b9abf).